packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx

```diff
@@ -164,8 +166,8 @@ export class DefaultFrontstage extends FrontstageProvider {
                       labelKey="iTwinViewer:components.properties"
                       applicationData={{
                         iModelConnection: UiFramework.getIModelConnection(),
-                        rulesetId: "Default",
-                        contextId: UiFramework.getIModelConnection()?.contextId,
+                        projectId: UiFramework.getIModelConnection()?.contextId,
```

Comment:
i just want to confirm renaming contextId to projectId

Reply:
Confimed: https://github.com/imodeljs/viewer-components-react/blob/5605c1312418d988eb3a4958d20601ce13133b5f/packages/property-grid/src/components/PropertyGrid.tsx#L62